ticoniq/jobconiq
# Feature/navbar

# Pull Request: Add Navbar Component

## Description

This PR introduces a new `Navbar` component for the JobConiq website. The navbar includes navigation links, a logo, and responsive design elements to ensure a seamless user experience across different devices.

## Changes

- Added a new `Navbar` component.
- Included the logo and navigation links (`Find Jobs` and `Browse Companies`).
- Implemented responsive design for mobile and desktop views.
- Integrated TailwindCSS for styling.
- Added a button to toggle the mobile menu.
- Enhanced accessibility with ARIA attributes.

## Screenshots

### Desktop View
![Screenshot 2024-06-05 031727](https://github.com/ticoniq/jobconiq/assets/71973088/036aab4e-0502-4300-9924-f2430295164c)

### Mobile View
![Screenshot 2024-06-05 031746](https://github.com/ticoniq/jobconiq/assets/71973088/47dfb790-180d-4de7-8f13-529e723b7947)

## How to Test

1. Pull the latest changes from the `feature/navbar` branch.
2. Run the development server: `npm run dev`.
3. Navigate to the home page and verify the navbar is displayed correctly.
4. Resize the browser window to test the responsive design.
5. Click the menu button on the mobile view to ensure the menu toggles correctly.

## Checklist

- [x] Navbar displays correctly on the desktop.
- [x] Navbar displays correctly on mobile.
- [x] Navigation links work as expected.
- [x] Menu button toggles the menu on mobile.

## Related Issues

- Closes #15: Implement Navbar Component

## Additional Notes

- Future enhancements might include dropdown menus for navigation links.
- Consider adding more links as the project expands.

## Preview Link

[Preview the changes here](https://jobconiq-7yb2zl9iy-ticoniqs-projects.vercel.app)

## Reviewer Notes

- Ensure the navbar is accessible and responsive.
- Verify the links direct to the appropriate pages.
- Check for any visual or functional discrepancies.

Thank you for reviewing this PR! Please let me know if there are any questions or additional changes required.
